### ISSUE TITLE
"standard" repository format does not work

### ISSUE THREAD

**Issue by vmarkovtsev** [src-d/jgit-spark-connector#422]
## Expected Behavior
If the repository format is set to "standard" jgit-spark-connector scans for `.git` and processes them.

## Current Behavior
`.git` seems to be ignored
## Steps to Reproduce (for bugs)
TODO

## Context
We cannot process normal repositories without this feature.

## Your Environment (for bugs)
* Spark version: 2.2.0
* PySpark version (if using PySpark): 2.2.0
* jgit-spark-connector version: 0.7.0

**Comment by vmarkovtsev** [comment]
@campoy Sorry, this is a real blocker to update Vecino so don't expect it to work soon.

**Comment by vmarkovtsev** [comment]
@ajnavarro Please have a look, this blocks.

**Comment by ajnavarro** [comment]
@vmarkovtsev the repository is directly inside `/tmp/modelforge` ? if not, that path is wrong.

If repositories are inside folders inside `/tmp/modelforge` the path should be `/tmp/modelforge/*`

**Comment by ajnavarro** [comment]
@vmarkovtsev sorry, we are adding /**/* automatically at the end of the provided path. I was able to load the repository putting it inside a repos folder. Can you try that please?

**Comment by vmarkovtsev** [comment]
Yes, putting `modelforge` into a subdirectory in `/tmp` works. My objective is to extract a single repository given its path. Is it possible without creating and moving directories?

**Comment by ajnavarro** [comment]
jgit-spark-connector is not intended to process just one repository, so the use case of reading just one repository pointing to his path is not expected.

To read just one repository you must put the repository in a parent directory, and use that directory on spark to read the repo.
